taichi/ir/frontend_ir.h

```diff
@@ -75,11 +75,12 @@ class FrontendIfStmt : public Stmt {
 
 class FrontendPrintStmt : public Stmt {
  public:
-  Expr expr;
-  std::string str;
```

Comment:
BTW, do you know what `str` was for? Was it used as a formatting string in the print statement?

Reply:
It's the variable name, e.g.:
`print(x)`, then `str = "x"`.